tests/test_ghostty_zig_version_sync.sh

```
@@ -83,107 +86,9 @@ if ! awk '
-
-validated_setup_jobs=0
-for workflow in "${tui_workflows[@]}"; do
-  job_count="$(validate_setup_zig_jobs "$workflow")"
-  validated_setup_jobs=$((validated_setup_jobs + job_count))
-done
-if [[ "$validated_setup_jobs" -eq 0 ]]; then
-  echo "No TUI setup-zig jobs were validated" >&2
-  exit 1
-fi
-
 if grep -Fq 'run: echo "version=$(bash ./scripts/ghostty-zig-version.sh)"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)